pelican-bootstrap3/templates/includes/twitter_timeline.html

```diff
@@ -2,9 +2,9 @@
 
     <li class="list-group-item"><h4><i class="fa fa-twitter fa-lg"></i><span class="icon-label">Latest Tweets</span></h4></li>
     <div id="twitter_timeline">
-        <a class="twitter-timeline" data-chrome="noheader" href="https://twitter.com/{{ TWITTER_USERNAME }}" data-widget-id="{{TWITTER_WIDGET_ID}}">Tweets by {{TWITTER_USERNAME}}</a>
+        <a class="twitter-timeline" data-chrome="noheader" href="https://twitter.com/{{ TWITTER_USERNAME }}">Tweets by {{ TWITTER_USERNAME }}</a> <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
     </div>
 
 <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
 
-{% endif %}
\ No newline at end of file
+{% endif %}
```

Comment:
Hey, thanks!  This helped me modify my own theme to account for what I assume must be recent Twitter changes with regard to Widgets, etc.

One thing I did have to change though is I changed line 1 from ` {% if TWITTER_WIDGET_ID %}` to `{% if TWITTER_TIMELINE %}`